### PR TITLE
#16: Remove inline css

### PR DESCRIPTION
**Pull request by amir-elgayed**

### CODE REVIEW

File: server/static/bindata_assetfs.go [hunk 1]
```diff
@@ -10,10 +10,10 @@
 package static
 
 import (
-	"github.com/elazarl/go-bindata-assetfs"
 	"bytes"
 	"compress/gzip"
 	"fmt"
```
Comment (ghaiszaher) [inline server/static/bindata_assetfs.go:15]: Looks unrelated

**Review by ghaiszaher — APPROVED** [review]

File: server/static/bindata_assetfs.go (hunk 1 above)
Comment (amir-elgayed) [reply server/static/bindata_assetfs.go:15]: @ghaiszaher I had to run a go command to bundle assets and it generates `bindata_assetfs.go` file to serve all assets, that's how this change come to be
